### PR TITLE
Thumbnail + new api + removed some unused imports.

### PR DESCRIPTION
**Pull request by sanderDijkxhoorn**
I have added the thumbnail image that VanMoof sends with the api to the bike selector, I think we should use some automated background remover or do it manually and make it use our custom images instead of these since the background is not transparent but some sort of white which also wouldn't match if we are adding a dark mode version of the app. I have changed the api urls to the ones that are currently used inside the iOS VanMoof app.

### CODE REVIEW

File: app/lib/api.dart [hunk 1]
```diff
@@ -19,7 +17,7 @@ Future<ApiClient> authenticate(String email, String password) async {
       responseType: ResponseType.json,
       receiveDataWhenStatusError: true,
     ));
-    resp = await client.get('https://my.vanmoof.com/api/v8/authenticate');
+    resp = await client.get('https://api.vanmoof-api.com/v8/authenticate');
```
Comment (mjarkk) [inline app/lib/api.dart:20]: Have you tested this?

Comment (sanderDijkxhoorn) [reply app/lib/api.dart:20]: Yes :)

File: app/lib/widgets/home_bikes_view/bike_view.dart [hunk 2]
```diff
@@ -56,6 +56,11 @@ class BikeView extends StatelessWidget {
                 ),
               ],
             ),
+            Image.network(
+              bike.links!.thumbnail,
+              fit: BoxFit.cover,
+              height: 200,
+            ),
```
Comment (mjarkk) [inline app/lib/widgets/home_bikes_view/bike_view.dart:63]: Can we remove this change (at least for now)

**Review by mjarkk — CHANGES_REQUESTED** [review]
Can you also update the web app to use the "new" api

File: app/lib/widgets/home_bikes_view/bike_view.dart (hunk 2 above)
Comment (sanderDijkxhoorn) [reply app/lib/widgets/home_bikes_view/bike_view.dart:63]: > 

Done :)

**Comment by sanderDijkxhoorn** [comment]
> Can you also update the web app to use the "new" api

Done :)

**Review by mjarkk — APPROVED** [review]

**Comment by sanderDijkxhoorn** [comment]
Oh I didn't remove the image yet 😅

**Comment by mjarkk** [comment]
Ah thanks for notifying then i'll remove it myself :)

**Pull request merged**